framework/common/theme/Color.kt

```diff
@@ -66,9 +68,12 @@ object Color {
             val relation: Color,
             val attribute: Color,
         )
+        override fun toString(): String {
+            return this.name
+        }
     }
 
-    private object DarkPalette {
+    private object CommonPalette {
```

Comment:
We've introduced a common palette since so many of the colours are shared between `DARK_GRAPH` and `LIGHT_GRAPH`.